generated/niscope/niscope/measurement_stats.py

```diff
@@ -17,7 +17,9 @@ def __repr__(self):
             'stdev={}'.format(self.stdev),
             'min_val={}'.format(self.min_val),
             'max_val={}'.format(self.max_val),
-            'num_in_stats={}'.format(self.num_in_stats)
+            'num_in_stats={}'.format(self.num_in_stats),
+            'channel={}'.format(self.channel),
+            'record={}'.format(self.record)
```

Comment:
Consulted with @thejcannon - `__repr__` should not include objects that are not part of class constructor.

Reply:
Got it. Reverting those changes.